tests/e2e/specs/shopper/filter-products-by-rating.test.ts

```diff
@@ -54,7 +57,7 @@ const goToShopPage = () =>
 		waitUntil: 'networkidle0',
 	} );
 
-describe.skip( `${ block.name } Block`, () => {
```

Comment:
Are we sure we wanna re-enable this test here? Based on https://github.com/woocommerce/woocommerce-blocks/issues/7744 it sounds like we are waiting on the next release of Gutenberg to do so.

Reply:
We skip tests related to `deleteAllTemplates` so it doesn't conflict with #7744. Although, we should update there to inform people about this.

Reply:
I noticed this, and I disabled only [the test regarding the PHP Classic Template](https://github.com/woocommerce/woocommerce-blocks/pull/7792/files#diff-b4f328a7e8563cddcfd31dfe080cb3a56e59e2dbef2f92ad24a7bc402810a3d9R97) (the only one that requires `deleteAllTemplates` function).

My idea is to open a PR for doing the same thing for other filters too.

Reply:
Sounds good! I also agree with @dinhtungdu 's suggestion of registering this info on the relevant issue for our future reference.